src/app/courses/courses.module.ts

```diff
@@ -23,7 +29,13 @@ import { MaterialModule } from '../shared/material.module';
   declarations: [
     CoursesComponent,
     CoursesAddComponent,
-    CoursesRequestComponent
-  ]
+    CoursesRequestComponent,
+    CourseOpenComponent,
+    CourseManageComponent,
+    CourseViewComponent,
+    CourseCreditsComponent,
+    CourseProgressComponent
+  ],
+  providers: [ ValidatorService ]

```

Comment:
Please remove this & the import statement from this file.  The `ValidatorService` is included in `app.module.ts` and does not need to be included as a provider in any of the other modules.

Reply:
@paulbert https://github.com/ole-vi/planet/pull/259/commits/c95c8c73d1d139a251dc9f0f7d506caba2328e5a